src/main.rs

```diff
@@ -188,7 +228,11 @@ async fn main() -> anyhow::Result<()> {
             .arg(Arg::with_name("cargo-args")
                 .multiple(true)
                 .last(true)
-                .help("Arguments to pass to the underlying `cargo build-bpf` command")))
+                .help("Arguments to pass to the underlying `cargo build-bpf` command"))
+            .arg(Arg::with_name("skip-build")
+                .long("skip-build")
+                .help("Skip building and verification, only upload the PDA")
+                .takes_value(false)))
```

Comment:
This is what `solana-verify remote submit-job` is for

Reply:
Okay this makes sense.

verify-from-repo --remote will now trigger write to PDA & submit job, skipping build